### DISCUSSION TITLE
NVidia overlay in-game

### DISCUSSION THREAD

**Discussion by SzymonOwczarek** [mmtrt/leagueoflegends#193] [Q&A]
Hi!

Is there any way to disable the Nvidia overlay displaying the:
-card type,
-drivers version,
-vulkan version,
-FPS?

Unfortunately, i can't screenshot of league of legends so can't provide a picture, but I'm sure somebody here knows!

**Comment by mmtrt** [comment]
See https://github.com/mmtrt/leagueoflegends/issues/183#issuecomment-907680906
